metrics-operator/converter/common.go

```diff
@@ -0,0 +1,22 @@
+package converter
+
+import "gopkg.in/inf.v0"
+
+const InvalidOperatorErrMsg = "invalid operator: '%s'"
+const UnableConvertValueErrMsg = "unable to convert value '%s' to decimal"
+const UnsupportedIntervalCombinationErrMsg = "unsupported interval combination '%s'"
+const EmptyOperatorsErrMsg = "empty operators: '%v'"
+const UnconvertableOperatorsCombinationErrMsg = "unconvertable combination of operators: '%s', '%s'"
+
+const MaxInt = int(^uint(0) >> 1)
```

Comment:
why not using `math.MaxInt64` directly?  https://pkg.go.dev/math#pkg-constants

Reply:
this should be arch independent (32 vs 64 bit architectures). I am not sure if maxInt64 has the same behaviour

Reply:
if you want architecture-independent, you can use `math.MaxInt` :) but we build only for 64bit: https://github.com/keptn/lifecycle-toolkit/blob/main/.github/workflows/release.yml#L180